service/k8sclient/k8sclient.go

```diff
@@ -1214,3 +1215,47 @@ func (c *K8sClient) checkOperatorStatus(installedVersions []string, expectedAPIV
 	}
 	return OperatorStatusNotInstalled
 }
+
+// GetClusterPods returns list of cluster's pods. It finds them by given cluster
+// name.
+func (c *K8sClient) GetClusterPods(ctx context.Context, clusterName string) (*common.PodList, error) {
+	list := &common.PodList{}
```

Comment:
let's keep using `new`

Reply:
Ok, but I am confused. Last time I used, `new` I was told not to use it. I guess it was in another repo where not using `new` was more consistent then using it.

Reply:
Done

Reply:
Yeah, I know and we discussed it in pmm-go-backend. We will keep using `new` in `dbaas-controller` for now.
To start using `&` we have to update all places where we use `new`